policy-manager/policy-manager_test.go

```
@@ -87,6 +87,30 @@ func TestPolicyManager(t *testing.T) {
+
+		regoContents := []byte(`package compliance_framework.set_violation
+
+import future.keywords.contains
+import future.keywords.if
+
+violation contains {"title": "Violation 1"} if {
+	input.violated
+}
+`)
+
+		var data map[string]interface{} = make(map[string]interface{})
```

golangci-lint (2.12.2)

[error] ST1023: should omit type map[string]interface{} from declaration; it will be inferred from the right-hand side

(staticcheck)